via/db.py

```diff
@@ -0,0 +1,57 @@
+import alembic.command
```

Comment:
This whole `db.py` file is from the cookiecutter's DB PR (https://github.com/hypothesis/cookiecutters/pull/128) which I don't think we're going to merge, so it can be considered manually added to Via